Mac silicon issue to load JNA library.

For some combinations of JDK and Mac silicon architecture this error happens during the plugin build.

```script
  Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.AssertionError: JNA library is not available [in thread "AWT-EventQueue-0"]
```

There is a workaround as the issue happens when adding a specific JNA from the license checker module. So it can be prevented adding the exclusion to the gradle file 

```script
  implementation("com.vaadin:license-checker:1.13.3") {
    exclude(group = "net.java.dev.jna", module = "jna")
    exclude(group = "net.java.dev.jna", module = "jna-platform")
  }
```

An example of validation tests failing (testExistingFile_multipleUndo) 

```script
java.lang.NoClassDefFoundError: Could not initialize class com.intellij.ui.components.MacScrollBarUI$Style
	at com.intellij.ui.components.MacScrollBarUI.installUI(MacScrollBarUI.java:86)
	at java.desktop/javax.swing.JComponent.setUI(JComponent.java:733)
	at java.desktop/javax.swing.JScrollBar.setUI(JScrollBar.java:218)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollBar.setUI(EditorImpl.java:3267)
	at com.intellij.ui.components.JBScrollBar.updateUI(JBScrollBar.java:87)
	at java.desktop/javax.swing.JScrollBar.<init>(JScrollBar.java:173)
	at com.intellij.ui.components.JBScrollBar.<init>(JBScrollBar.java:65)
	at com.intellij.ui.components.JBScrollBar.<init>(JBScrollBar.java:61)
	at com.intellij.openapi.editor.impl.EditorImpl$OpaqueAwareScrollBar.<init>(EditorImpl.java:3223)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollBar.<init>(EditorImpl.java:3256)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollPane.createVerticalScrollBar(EditorImpl.java:5413)
	at java.desktop/javax.swing.JScrollPane.<init>(JScrollPane.java:301)
	at java.desktop/javax.swing.JScrollPane.<init>(JScrollPane.java:353)
	at com.intellij.ui.components.JBScrollPane.<init>(JBScrollPane.java:83)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollPane.<init>(EditorImpl.java:5350)
	at com.intellij.openapi.editor.impl.EditorImpl.<init>(EditorImpl.java:360)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor(EditorFactoryImpl.java:222)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.java:203)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.java:158)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.doOpenTextEditor(TestEditorManagerImpl.kt:468)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileImpl3(TestEditorManagerImpl.kt:159)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileInCommand$lambda$6(TestEditorManagerImpl.kt:448)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:215)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:176)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:166)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:152)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileInCommand(TestEditorManagerImpl.kt:447)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openTextEditor(TestEditorManagerImpl.kt:437)
	at com.vaadin.plugin.copilot.handler.AbstractHandler.openFileInEditor(AbstractHandler.kt:76)
	at com.vaadin.plugin.copilot.handler.AbstractHandler.postSave(AbstractHandler.kt:71)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler.writeAndFlush$lambda$2$lambda$1$lambda$0(WriteFileHandler.kt:63)
	at com.intellij.openapi.command.WriteCommandAction.lambda$runWriteCommandAction$4(WriteCommandAction.java:338)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$1(WriteCommandAction.java:144)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:975)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$2(WriteCommandAction.java:142)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:215)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:186)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.doRunWriteCommandAction(WriteCommandAction.java:151)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.run(WriteCommandAction.java:118)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:338)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:326)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler.writeAndFlush$lambda$2$lambda$1(WriteFileHandler.kt:61)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:224)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:176)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:166)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler.writeAndFlush(WriteFileHandler.kt:58)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler.access$writeAndFlush(WriteFileHandler.kt:19)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler$run$1.invoke(WriteFileHandler.kt:33)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler$run$1.invoke(WriteFileHandler.kt:31)
	at com.intellij.openapi.application.ActionsKt.runInEdt(actions.kt:56)
	at com.intellij.openapi.application.ActionsKt.runInEdt$default(actions.kt:53)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler.run(WriteFileHandler.kt:31)
	at com.vaadin.plugin.copilot.service.CopilotUndoManagerTest$callFileWriteHandler$$inlined$runInEdtAndWait$1.run(EdtTestUtil.kt:43)
	at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$5(EdtTestUtil.java:95)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:209)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:22)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:191)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:843)
	at com.intellij.openapi.application.impl.ApplicationImpl$4.run(ApplicationImpl.java:463)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:101)
	at com.intellij.openapi.application.impl.RwLockHolder.runWithEnabledImplicitRead(RwLockHolder.kt:75)
	at com.intellij.openapi.application.impl.RwLockHolder.runWithImplicitRead(RwLockHolder.kt:67)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWithImplicitRead(ApplicationImpl.java:1430)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:82)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:124)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:44)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:792)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:695)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$12(IdeEventQueue.kt:589)
	at com.intellij.openapi.application.impl.RwLockHolder.runWithoutImplicitRead(RwLockHolder.kt:44)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:589)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:72)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:355)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:354)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:793)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:354)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1014)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:114)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1014)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:349)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:848)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:391)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.AssertionError: JNA library is not available [in thread "AWT-EventQueue-0"]
	at com.intellij.ui.mac.foundation.Foundation.<clinit>(Foundation.java:30)
	at com.intellij.ui.mac.foundation.Foundation$NSAutoreleasePool.<init>(Foundation.java:491)
	at com.intellij.ui.components.MacScrollBarUI.callMac(MacScrollBarUI.java:181)
	at com.intellij.ui.components.MacScrollBarUI$Native.<init>(MacScrollBarUI.java:279)
	at com.intellij.ui.components.MacScrollBarUI$Style$1.<init>(MacScrollBarUI.java:231)
	at com.intellij.ui.components.MacScrollBarUI$Style.<clinit>(MacScrollBarUI.java:231)
	at com.intellij.ui.components.MacScrollBarUI.installUI(MacScrollBarUI.java:86)
	at java.desktop/javax.swing.JComponent.setUI(JComponent.java:733)
	at java.desktop/javax.swing.JScrollBar.setUI(JScrollBar.java:218)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollBar.setUI(EditorImpl.java:3267)
	at com.intellij.ui.components.JBScrollBar.updateUI(JBScrollBar.java:87)
	at java.desktop/javax.swing.JScrollBar.<init>(JScrollBar.java:173)
	at com.intellij.ui.components.JBScrollBar.<init>(JBScrollBar.java:65)
	at com.intellij.ui.components.JBScrollBar.<init>(JBScrollBar.java:61)
	at com.intellij.openapi.editor.impl.EditorImpl$OpaqueAwareScrollBar.<init>(EditorImpl.java:3223)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollBar.<init>(EditorImpl.java:3256)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollPane.createVerticalScrollBar(EditorImpl.java:5413)
	at java.desktop/javax.swing.JScrollPane.<init>(JScrollPane.java:301)
	at java.desktop/javax.swing.JScrollPane.<init>(JScrollPane.java:353)
	at com.intellij.ui.components.JBScrollPane.<init>(JBScrollPane.java:83)
	at com.intellij.openapi.editor.impl.EditorImpl$MyScrollPane.<init>(EditorImpl.java:5350)
	at com.intellij.openapi.editor.impl.EditorImpl.<init>(EditorImpl.java:360)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor(EditorFactoryImpl.java:222)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.java:203)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.java:158)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.doOpenTextEditor(TestEditorManagerImpl.kt:468)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileImpl3(TestEditorManagerImpl.kt:159)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileInCommand$lambda$6(TestEditorManagerImpl.kt:448)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:215)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:176)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:166)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:152)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileInCommand(TestEditorManagerImpl.kt:447)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openTextEditor(TestEditorManagerImpl.kt:437)
	at com.vaadin.plugin.copilot.handler.AbstractHandler.openFileInEditor(AbstractHandler.kt:76)
	at com.vaadin.plugin.copilot.handler.AbstractHandler.postSave(AbstractHandler.kt:71)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler$create$1.invoke$lambda$1$lambda$0(WriteFileHandler.kt:81)
	at com.intellij.openapi.command.WriteCommandAction.lambda$runWriteCommandAction$4(WriteCommandAction.java:338)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$1(WriteCommandAction.java:144)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:975)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.lambda$doRunWriteCommandAction$2(WriteCommandAction.java:142)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:215)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:186)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.doRunWriteCommandAction(WriteCommandAction.java:151)
	at com.intellij.openapi.command.WriteCommandAction$BuilderImpl.run(WriteCommandAction.java:118)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:338)
	at com.intellij.openapi.command.WriteCommandAction.runWriteCommandAction(WriteCommandAction.java:326)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler$create$1.invoke$lambda$1(WriteFileHandler.kt:79)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:224)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:176)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:166)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler$create$1.invoke(WriteFileHandler.kt:76)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler$create$1.invoke(WriteFileHandler.kt:74)
	at com.intellij.openapi.application.ActionsKt.runInEdt(actions.kt:56)
	at com.intellij.openapi.application.ActionsKt.runInEdt$default(actions.kt:53)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler.create(WriteFileHandler.kt:74)
	at com.vaadin.plugin.copilot.handler.WriteFileHandler.run(WriteFileHandler.kt:41)
	... 43 more
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -46,7 +46,10 @@ repositories {
 dependencies {
   implementation("com.amplitude:java-sdk:[1.8.0,2.0)")
   implementation("org.json:json:20201115")
-  implementation("com.vaadin:license-checker:1.13.3")
+  implementation("com.vaadin:license-checker:1.13.3") {
+    exclude(group = "net.java.dev.jna", module = "jna")
+    exclude(group = "net.java.dev.jna", module = "jna-platform")
+  }
 
   intellijPlatform {
     intellijIdeaUltimate(buildVersion)

